src/vs/editor/common/core/selection.ts

```diff
@@ -128,6 +128,12 @@ export class Selection extends Range {
 		return new Position(this.positionLineNumber, this.positionColumn);
 	}
 
+	/**
+	 * Create a position using startLineNumber and startColumn as the start position.
+	 */
+	public setPositionFromStart(startLineNumber: number, startColumn: number): Position {
+		return new Position(startLineNumber, startColumn);
```

Comment:
I don't think this is needed as you can use `new Position(...)`

Reply:
same as above, removed entirely!